main.tf

```diff
@@ -200,12 +200,63 @@ resource "aws_elasticache_replication_group" "global" {
   }
 }
 
+################################################################################
+# Serverless Cache
+################################################################################
+
+resource "aws_elasticache_serverless_cache" "this" {
```

Comment:
should this be in the main module, or its own sub-module? Other than the security group, I don't see any other resources from `main.tf` pulled in here which makes me think its better if its in its own sub-module